app/assets/stylesheets/_sn.scss

```
@@ -1,322 +1,17 @@
-}
-
-.mr-10 {
-  margin-right: 2.5rem;
-}
-
-.-mt-1 {
-  margin-top: -0.25rem;
-}
-
-.-mr-1 {
-  margin-right: -0.25rem;
-}
-
-.-mr-2 {
-  margin-right: -0.5rem;
-}
-
-.py-1 {
-  padding-top: 0.25rem;
-  padding-bottom: 0.25rem;
-}
-
-.pl-1 {
-  padding-left: 0.25rem;
-}
-
-.pr-2 {
-  padding-right: 0.5rem;
-}
-
-.px-1 {
-  padding-left: 0.25rem;
-  padding-right: 0.25rem;
-}
-
-.px-2 {
-  padding-left: 0.5rem;
-  padding-right: 0.5rem;
-}
-
-.py-1\.5 {
-  padding-top: 0.375rem;
-  padding-bottom: 0.375rem;
-}
-
-.py-8 {
-  padding-top: 2rem;
-  padding-bottom: 2rem;
-}
-
-.outline-none {
-  outline: none;
-}
-
-.color-foreground {
-  color: var(--sn-stylekit-foreground-color);
-}
-
-.color-danger {
-  color: var(--sn-stylekit-danger-color);
-}
-
-.color-info {
-  color: var(--sn-stylekit-info-color);
-}
-
-.ring-info {
-  box-shadow: 0 0 0 2px var(--sn-stylekit-info-color);
-}
-
-.inner-ring-info {
-  box-shadow: inset 0 0 0 2px var(--sn-stylekit-info-color);
-}
-
-.focus\:bg-contrast:focus {
-  @extend .bg-contrast;
-}
-
-.hover\:color-text:hover {
-  @extend .color-text;
-}
-
-.focus\:color-text:focus {
-  @extend .color-text;
-}
-
-.hover\:bg-secondary-contrast:hover {
-  @extend .bg-secondary-contrast;
-}
-
-.focus\:bg-secondary-contrast:focus {
-  @extend .bg-secondary-contrast;
-}
-
-.focus\:inner-ring-info:focus {
-  @extend .inner-ring-info;
-}
-
-.focus\:ring-info:focus {
-  @extend .ring-info;
-}
-
-.focus-within\:ring-info:focus-within {
-  @extend .ring-info;
-}
-
-.text-left {
-  text-align: left;
-}
-
-.text-3xl {
-  font-size: 1.875rem;
-  line-height: 2.25rem;
-}
-
-.w-0 {
-  width: 0;
-}
-
-.w-3\.5 {
-  width: 0.875rem;
-}
-
-.w-5 {
-  width: 1.25rem;
-}
-
-.w-8 {
-  width: 2rem;
-}
-
-.max-w-290px {
-  max-width: 290px;
-}
-
-.max-w-xs {
-  max-width: 20rem;
-}
-
-.max-w-40 {
-  max-width: 10rem;
-}
-
-.min-w-5 {
-  min-width: 1.25rem;
-}
-
-.min-w-40 {
-  min-width: 10rem;
-}
-
-.h-1px {
-  height: 1px;
-}
-
-.h-0 {
-  height: 0;
-}
-
-.h-3\.5 {
-  height: 0.875rem;
-}
-
-.h-4\.5 {
-  height: 1.125rem;
-}
-
-.h-5 {
-  height: 1.25rem;
-}
-
-.h-6 {
-  height: 1.5rem;
-}
-
-.h-7 {
-  height: 1.75rem;
-}
-
-.h-8 {
-  height: 2rem;
-}
-
-.h-9 {
-  height: 2.25rem;
-}
-
-.h-10 {
-  height: 2.5rem;
-}
-
-.h-18 {
-  height: 4.5rem;
-}
-
-.h-screen {
-  height: 100vh;
-}
-
-.max-h-120 {
-  max-height: 30rem;
-}
-
-.min-h-5 {
-  min-height: 1.25rem;
-}
-
-.fixed {
-  position: fixed;
-}
-
-.overflow-y-scroll {
-  overflow-y: scroll;
-}
-
-.overflow-auto {
-  overflow: auto;
-}
-
-.overflow-hidden {
-  overflow: hidden;
-}
-
-.overflow-ellipsis {
-  text-overflow: ellipsis;
-}
-
-.items-start {
-  align-items: flex-start;
-}
-
-.p-2 {
-  padding: 0.5rem;
-}
-
-.flex-wrap {
-  flex-wrap: wrap;
-}
-
-.whitespace-pre-wrap {
-  white-space: pre-wrap;
-}
-
-.whitespace-nowrap {
-  white-space: nowrap;
-}
-
-.w-80 {
-  width: 20rem;
-}
-
-.w-70 {
-  width: 17.5rem;
+.hover-text-underline {
+  &:hover {
+    text-decoration: underline;
+  }
```

```suggestion
.hover\:underline:hover {
  text-decoration: underline;
}
```

To follow the naming conventions we currently use on StyleKit 🙂

---

Noted, thanks!